app/navigation/ReturnsJourneyNavigator.scala

```diff
@@ -76,14 +76,19 @@ class ReturnsJourneyNavigator {
       case _ => throw new Exception("Unable to navigate to page")
     }
 
-  def manufacturedPlasticPackagingRoute(mode: Mode, hasAnswerChanged: Boolean, usersAnswer: Boolean): Call = 
-    if (hasAnswerChanged) 
-      (mode, usersAnswer) match {
-        case (_, true)       => routes.ManufacturedPlasticPackagingWeightController.onPageLoad(mode)
-        case (CheckMode, _)  => routes.ConfirmPlasticPackagingTotalController.onPageLoad
-        case (NormalMode, _) => routes.ImportedPlasticPackagingController.onPageLoad(mode)
-    } else 
-      routes.ConfirmPlasticPackagingTotalController.onPageLoad
+  def manufacturedPlasticPackagingRoute(mode: Mode, hasAnswerChanged: Boolean, usersAnswer: Boolean): Call = {
+    if(mode.equals(NormalMode)) {
+      if(usersAnswer)
+        routes.ManufacturedPlasticPackagingWeightController.onPageLoad(mode)
+      else
+        routes.ImportedPlasticPackagingController.onPageLoad(mode)
+    }else {
+      if(usersAnswer && hasAnswerChanged)
+        routes.ManufacturedPlasticPackagingWeightController.onPageLoad(mode)
+      else
+        routes.ConfirmPlasticPackagingTotalController.onPageLoad
+    }
```

Comment:
Could be written as;


```
    (mode.equals(NormalMode), usersAnswer, hasAnswerChanged) match {
      case (true, true, _)      => routes.ManufacturedPlasticPackagingWeightController.onPageLoad(mode)
      case (true, false, _)     => routes.ImportedPlasticPackagingController.onPageLoad(mode)
      case (false, true, true)  => routes.ManufacturedPlasticPackagingWeightController.onPageLoad(mode)
      case (false, _, _)        => routes.ConfirmPlasticPackagingTotalController.onPageLoad
    }
```

Reply:
Or 

 ```
private def startYourReturnRoute(answers: UserAnswers): Call =
    answers.get(StartYourReturnPage) match {
      case Some(true) => routes.ManufacturedPlasticPackagingController.onPageLoad(NormalMode)
      case Some(false) => routes.NotStartOtherReturnsController.onPageLoad()
      case _ => throw new Exception("Unable to navigate to page")
    }

  private def manufacturedPlasticPackagingRouteNormal(usersAnswer: Boolean): Call = {
    if (usersAnswer) {
      routes.ManufacturedPlasticPackagingWeightController.onPageLoad(Mode.NormalMode)
    } else {
      routes.ImportedPlasticPackagingController.onPageLoad(Mode.NormalMode)
    }
  }

  private def manufacturedPlasticPackagingRouteCheck(hasAnswerChanged: Boolean, usersAnswer: Boolean): Call = {
    if(usersAnswer && hasAnswerChanged) {
      routes.ManufacturedPlasticPackagingWeightController.onPageLoad(Mode.CheckMode)
    } else {
      routes.ConfirmPlasticPackagingTotalController.onPageLoad
    }
  }
```

Reply:
yep. I think the last option is good. Thanks